src/main/java/de/azapps/kafkabackup/sink/BackupSinkTask.java

```diff
@@ -69,15 +124,26 @@ public void put(Collection<SinkRecord> records) {
                 if (sinkRecord.kafkaOffset() % 100 == 0) {
                     log.debug("Backed up Topic {}, Partition {}, up to offset {}", sinkRecord.topic(), sinkRecord.kafkaPartition(), sinkRecord.kafkaOffset());
                 }
+                if (config.snapShotMode()) {
+                    currentOffsets.put(topicPartition, sinkRecord.kafkaOffset());
+                    log.debug("Update offset for topic {}, partition {}, offset {}", sinkRecord.topic(), sinkRecord.kafkaPartition(), sinkRecord.kafkaOffset());
```

Comment:
I think this makes sense to log-debug for all `put()` calls and not just for snapshots – or do not do that at all ;)

Reply:
Removed it